nextcord/enums.py

```diff
@@ -177,6 +178,16 @@ def try_value(cls, value):
             except (KeyError, TypeError):
                 return value
 
+        @property
+        def name(self):
+            """The name of the Enum member."""
+            return self.name
+
+        @property
+        def value(self):
```

Comment:
@DenverCoder1  
## Chech this... 
### It can fix pycharm warnings like following:
![image](https://user-images.githubusercontent.com/39167433/165431153-5f3543dd-d157-49dd-be06-2a82b49836e8.png)
![image](https://user-images.githubusercontent.com/39167433/165431210-92db961b-1039-4838-a0e2-bff981371b58.png)

Reply:
How is this related to the pr?
Type issues should target other prs

Reply:
this was not resolved, why was it marked as so

if you believe it is resolved, tell me why, otherwise dont blank me

Reply:
Ooo sorry, I didn't notice it...
You are right. That is not related to this pr... Should I revert the changes or let it be like that? 
It actually fixes warnings though.

Reply:
This change should be reverted and put into a new pull request which is specifically for resolving these kinds of issues